backend/tests/test_docker_sim.py

```
@@ -44,121 +38,95 @@ async def check_health():
-    )
+def test_service_logs_accessible():
+    """Test that service log files are accessible"""
+    import os
+
+    # Check if log directory exists (it should be created by setup)
+    log_dir = "./logs"
+    if not os.path.exists(log_dir):
+        pytest.skip("Log directory not found - run setup_logs.sh first")
+
+    # Check for validator log file
+    validator_log = os.path.join(log_dir, "validator.log")
```

Ruff (0.12.2)

Local variable `validator_log` is assigned to but never used

Remove assignment to unused variable `validator_log`

(F841)